### ISSUE TITLE
When parsing of the file fails, it should at least output the name of the file that cannot be parsed

### ISSUE THREAD

**Issue by capaj** [lebab/lebab#327]
I get output like this when running in a folder:
```
lebab --replace "lib/**/*.js" --transform commonjs
/home/capaj/.nvm/versions/node/v14.17.4/lib/node_modules/lebab/node_modules/espree/lib/espree.js:195
            throw err;
            ^

SyntaxError: Unexpected token .
    at Espree.raise (/home/capaj/.nvm/versions/node/v14.17.4/lib/node_modules/lebab/node_modules/espree/lib/espree.js:190:25)
    at Espree.unexpected (/home/capaj/.nvm/versions/node/v14.17.4/lib/node_modules/lebab/node_modules/espree/lib/espree.js:235:18)
    at Espree.pp$3.parseExprAtom (/home/capaj/.nvm/versions/node/v14.17.4/lib/node_modules/lebab/node_modules/acorn/dist/acorn.js:2326:12)
    at Espree.parseExprAtom (/home/capaj/.nvm/versions/node/v14.17.4/lib/node_modules/lebab/node_modules/acorn-jsx/index.js:445:22)
    at Espree.pp$3.parseExprSubscripts (/home/capaj/.nvm/versions/node/v14.17.4/lib/node_modules/lebab/node_modules/acorn/dist/acorn.js:2129:21)
    at Espree.pp$3.parseMaybeUnary (/home/capaj/.nvm/versions/node/v14.17.4/lib/node_modules/lebab/node_modules/acorn/dist/acorn.js:2106:19)
    at Espree.pp$3.parseExprOps (/home/capaj/.nvm/versions/node/v14.17.4/lib/node_modules/lebab/node_modules/acorn/dist/acorn.js:2041:21)
    at Espree.pp$3.parseMaybeConditional (/home/capaj/.nvm/versions/node/v14.17.4/lib/node_modules/lebab/node_modules/acorn/dist/acorn.js:2024:21)
    at Espree.pp$3.parseMaybeAssign (/home/capaj/.nvm/versions/node/v14.17.4/lib/node_modules/lebab/node_modules/acorn/dist/acorn.js:1997:21)
    at Espree.pp$3.parseMaybeConditional (/home/capaj/.nvm/versions/node/v14.17.4/lib/node_modules/lebab/node_modules/acorn/dist/acorn.js:2029:30) {
  index: 529,
  lineNumber: 15,
  column: 24
}

```

I think there is some file which cannot be parsed by espree, but it's very hard to know which one.

**Comment by nene** [comment]
This is now fixed in 3.2.1 release.

**Comment by capaj** [comment]
Amazing, thanks!
